docs/tools.md

```diff
@@ -45,7 +45,7 @@ See following example. The RefArch templates are already configured this way.
 
 The provided [Renovate](https://renovatebot.com) configuration [`refarch-renovate-config.json5`](https://github.com/it-at-m/refarch/blob/main/refarch-tools/refarch-renovate/refarch-renovate-config.json5) extends the [it@M global Renovate configurations](https://github.com/it-at-m/.github/tree/main/renovate-configs).
 Additionally, it includes specific presets for tagging and dependency restrictions to prevent Renovate from suggesting updates to dependencies that are pinned by the RefArch.
-The configuration also enables auto-merging of patch releases and pinning operations for GitHub Actions, NPM dependencies, Maven artifacts and Dockerfile base images while keeping pinning of dependencies intact.
+The configuration also enables auto-merging of patch releases and pinning operations for GitHub Actions, NPM dependencies, Maven artifacts, image versions in `docker-compose.yml` and Dockerfile base images while keeping pinning of dependencies intact.
```

Comment:
"and pinning operations" ... "while keeping pinning of dependencies intact." seems a bit duplicated.
Also the listed articats are just examples and not restricted to them so I would prefer a "e.g." somewhere.